about.html

```diff
@@ -37,7 +37,7 @@
           <a href="">&times;</a>
           <a href="index.html" class="close-menu">Home</a>
           <a href="about.html" class="close-menu">About Us</a>
-          <a href="#" class="close-menu">Inquirires</a>
+          <a href="" class="close-menu">Inquirires</a>
           <a href="#" class="close-menu">Partner</a>
        </div>
```

Comment:
- [ ] The Header section in the About page should contain a line with contact information or any other prompt under the paragraph, kindly consider implementing this so your website looks identical to the provided design

|Your website|Provided design|
|-|-|
|![Screenshot_20230706_201818](https://github.com/paulshaun-kdtk/capstone-Car-Show/assets/25227022/22b2d535-20f4-40f2-9888-7b652552d95f)|![Screenshot_20230706_201922](https://github.com/paulshaun-kdtk/capstone-Car-Show/assets/25227022/68684ec1-5c51-4342-8486-96680e21217e)|